dev/generator/main.go

```diff
@@ -75,9 +76,9 @@ func Build(sourceDir, publicDir string) error {
 
 // CopyPackage copies the files of a package to the public directory
 func CopyPackage(src, dst string) error {
-	fmt.Println(">> Copy package: " + src)
+	log.Println(">> Copy package: " + src)
 	os.MkdirAll(dst, 0755)
-	err := sh.RunV("cp", "-a", src, dst)
+	err := sh.RunV("rsync", "-a", src, dst)
```

Comment:
@kuisathaverat I switched to `rsync` to have better cross platform consistency. But it is not yet on CI. Is there an easy way to get it there?